webroot/js/toolbar-app.js

```diff
@@ -130,6 +131,17 @@ Toolbar.prototype = {
 	},
 
 	bindNeatArray: function() {
+        var sortButton = this.content.find('.neat-array-sort');
+        var _this = this;
+        sortButton.click(function() {
+            if ($(this).attr('checked')) {
+                document.cookie = 'debugKit_sort=; expires=Thu, 01 Jan 1970 00:00:01 GMT; path=' + _this.baseUrl;
+            } else {
+                document.cookie = 'debugKit_sort=1; path=' + _this.baseUrl;
+            }
+            _this.loadPanel(_this.currentPanel());
+        });
```

Comment:
The indentation is inconsistent. The rest of this file is tabs. There is probably a good case for converting the file over to spaces, but that can be done separately I guess.